controllers/sr_controller/controller.pyi

```diff
@@ -0,0 +1,7 @@
+# Type stubs for the Webots controller file
```

Comment:
Any way this file can be merged with https://github.com/srobo/competition-simulator/blob/master/script/testing/webots-fakes/controller.py?

Reply:
Ah, possibly.

Reply:
b0a5b1d19ba9f5164de024c8ca1ad088bc7140ed (see https://github.com/srobo/competition-simulator/pull/109)